actors/evm/Makefile

```diff
@@ -23,3 +25,54 @@ $(TEST_CONTRACTS_DIR)/%.hex: $(TEST_CONTRACTS_DIR)/%.sol | solc
 	@# We could use just the .bin files in the test, but this is a way to stick to the existing pattern.
 	solc --bin $< | tail -n +4 | tr -d '\n' > $@
 	solc --bin --abi --storage-layout --hashes --overwrite $< -o $(TEST_CONTRACTS_DIR)
+
+
+# Render measurement charts.
+.PHONY: plot-measurements
+plot-measurements: $(MEASUREMENTS_PNG)
+
+# Render a specfic plot if the data changed.
+$(MEASUREMENTS_DIR)/%.png: $(MEASUREMENTS_DIR)/%.jsonline $(MEASUREMENTS_DIR)/storage-footprint.plt | jq gnuplot
+
```

Comment:
I'd consider breaking this out into a separate script to make it easier to maintain.

Reply:
Done.